package.json

```diff
@@ -22,7 +22,9 @@
     "test": "vitest --project unit",
     "test:bun": "bun test --bail",
     "test:e2e": "vitest --project e2e",
-    "test:leaks": "cross-env \"LEAK_TEST=1\" jest --no-watchman --runInBand --detectOpenHandles --detectLeaks --logHeapUsage"
+    "test:leaks": "cross-env \"LEAK_TEST=1\" jest --no-watchman --runInBand --detectOpenHandles --detectLeaks --logHeapUsage",
+    "build:alpha": "yarn build && yarn pin-workspace-deps",
+    "pin-workspace-deps": "node ./scripts/pin-workspace-deps.mjs"
```

Comment:
You should write the scripts in typescript, add them to the `include` in tsconfig and execute with tsx. This way everything is kept typesafe.